test/ci_workflow_gen.go

```diff
@@ -31,17 +31,16 @@ const (
 	unitTestTemplate  = "templates/unit_test.tpl"
 	unitTestDatabases = "percona56, mysql57, mysql80, mariadb101, mariadb102, mariadb103"
 
-	clusterTestTemplate        = "templates/cluster_endtoend_test.tpl"
-	clusterList                = "11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 26, 27, vreplication_basic"
-	clustersRequiringMakeTools = "18,24"
-
+	clusterTestTemplate = "templates/cluster_endtoend_test.tpl"
+	clusterList         = "11,12,13,14,15,16,17,18,19,20,21,22,23,24,26,27,vreplication_basic,vreplication_multicell,vreplication_cellalias,vreplication_v2"
 	// TODO: currently some percona tools including xtrabackup are installed on all clusters, we can possibly optimize
 	// this by only installing them in the required clusters
 	clustersRequiringXtraBackup = clusterList
+	clustersRequiringMakeTools  = "18,24"
 )
 
 type unitTest struct {
-	Name, Database string
+	Name, Platform string
```

Comment:
I assume this change fixes the error we were getting from `make`:
```
Error: template: unit_test.tpl:22:17: executing "unit_test.tpl" at <.Platform>: can't evaluate field Platform in type *main.unitTest
```

Reply:
Yes,